lib/pathfinder.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+const Errors = require('./errors')
+
+const lifecycleStages = [
+  'configure',
```

Comment:
not sure. I think since `validate()` should not modify any application state, then it doesn't make sense to rely on other trailpacks in this phase. It's easy to add if we decide to change this.